sp800-63-3/sec4_model.md

```diff
@@ -88,7 +88,7 @@ A digital authentication system may incorporate multiple factors in one of two w
 1. The system may be implemented so that multiple factors are presented to the verifier; or
 2. Some factors may be used to protect a secret that will be presented to the verifier. 
 
-For example, consider a piece of hardware (the authenticator) that contains a cryptographic key (the authenticator secret) where access is protected with a fingerprint. When used with the biometric, the cryptographic key produces an output that is used in the authentication process to authenticate the claimant. An impostor must steal the encrypted key (by stealing the hardware) and replicate the fingerprint to use the authenticator. These guidelines consider such a device effective to provide MFA, although the actual authentication protocol between the verifier and the claimant simply proves possession of the key.
+For example, item 1 can be satisfied by pairing a memorized secret (what you know) with an out-of-band-device (what you have). Both authenticator outputs are presented to the verifier to authenticate the claimant. For item 2, consider a piece of hardware (the authenticator) that contains a cryptographic key (the authenticator secret) where access is protected with a fingerprint. When used with the biometric, the cryptographic key produces an output that is used in the authentication process to authenticate the claimant.
```

Comment:
nit: "out-of-band-device" -> "out-of-band device"

Possibly remove "in the authentication process" in last sentence. Seems redundant.